content/en/docs/installation/upgrading/upgrading-1.2-1.3.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Upgrading from v1.2 to v1.3"
+linkTitle: "v1.2 to v1.3"
+weight: 800
+type: "docs"
+---
+
+## Upgrade notes for users of the Venafi Cloud Issuer
+
+This release updates the [Venafi Cloud Issuer][] to use `OutagePREDICT` instead of `DevOpsACCELERATE`.
+
+The only impact to Venafi Cloud users is the change in zone syntax.
+The zone is now `<Application Name>\<Issuing Template Alias>`
+(e.g. `My Application\My CIT`).
+
+### Background
+
+Venafi are currently transitioning Venafi Cloud users to the `OutagePREDICT` ("OP") product,
+from `DevOpsACCELERATE` ("DA"), which will be sunset later in 2021.
+
+The [Venafi Cloud Issuer][] in cert-manager relies upon the `VCert` library,
+and the [`VCert` `v4.13.0`][] release marks this "DA2OP" transition.
+The `VCert` module dependencies in cert-manager have been updated in order for cert-manager to complete the transition as well.
+
+With this update, cert-manager users with Venafi Cloud issuers will need to be aware that the zone format changes from a UUID (DA Zone ID) to a string of the form `<Application Name>\<Issuing Template Alias>`.
```

Comment:
nit: to make sure people notice the new zone format, I propose to put it in a code block:

```suggestion
With this update, cert-manager users with Venafi Cloud issuers will need to be aware that the zone format changes from a UUID (DA Zone ID) to a string of the form:
```text
    <Application Name>\<Issuing Template Alias>
```
```